sycl/doc/GetStartedWithSYCLCompiler.md

```diff
@@ -1,111 +1,63 @@
 # Overview
+
 The SYCL* Compiler compiles C++\-based SYCL source files with code for both CPU
 and a wide range of compute accelerators. The compiler uses Khronos*
 OpenCL&trade; API to offload computations to accelerators.
 
+# Table of contents
 
-# Before You Begin
-
-### Get `OpenCL runtime` for CPU and/or GPU on `Linux`:
-
-   a. OpenCL&trade; runtime for GPU: follow instructions on
-[github.com/intel/compute-runtime/releases](https://github.com/intel/compute-runtime/releases)
-to install.
-
-   b. Experimental Intel&reg; CPU Runtime for OpenCL&trade; Applications with
-SYCL support: follow the instructions under
-[SYCL* Compiler and Runtimes](https://github.com/intel/llvm/releases/tag/2019-07)
-
-### Get `OpenCL runtime` for CPU and/or GPU on `Windows`:
-   a. OpenCL&trade; runtime for GPU and CPU: download it from
-   [Intel&reg; Download Center](https://downloadcenter.intel.com/download/28991/Intel-Graphics-Windows-10-DCH-Drivers)
+* [Prerequisites](#prerequisites)
+  * [Create SYCL workspace](#create-sycl-workspace)
+* [Build SYCL toolchain](#build-sycl-toolchain)
+  * [Build SYCL toolchain with libc++ library](#build-sycl-toolchain-with-libc-library)
+* [Use SYCL toolchain](#use-sycl-toolchain)
+  * [Install low level runtime](#install-low-level-runtime)
+  * [Test SYCL toolchain](#test-sycl-toolchain)
+  * [Run simple SYCL application](#run-simple-sycl-application)
+* [C++ standard](#c-standard)
+* [Known Issues and Limitations](#known-issues-and-limitations)
+* [Find More](#find-more)
 
-   b. The latest/experimental Intel&reg; CPU Runtime for OpenCL&trade; Applications with SYCL support
-   on Windows `will soon be published` together with Linux runtime at [SYCL* Compiler and Runtimes](https://github.com/intel/llvm/releases)
+# Prerequisites
 
-### Get the required tools:
+* `git` - https://git-scm.com/downloads
+* `cmake` version 3.2 or later - http://www.cmake.org/download/
+* `GCC` version 5.1.0 or later (including libstdc++) -
+  https://gcc.gnu.org/install/
+* `python` - https://www.python.org/downloads/release/python-2716/
+* `Visual Studio 2017` or later (Windows only) -
+   https://visualstudio.microsoft.com/downloads/
```

Comment:
Please specify exact version (MSVC 19.15.26730.0 and higher). We are observing build failures on earlier versions.

Reply:
Could you provide Visual Studio version, please?
https://docs.microsoft.com/en-us/visualstudio/releases/2019/release-notes

Reply:
We had no problems with MSVC2017 before. If It does not work we could fix it.
CLANG requires MSVC2017 or newer, why we would raise the bar?